.github/workflows/frontend-tests.yml

```diff
@@ -5,6 +5,12 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ '*' ]
+    paths:
+      - '**.jsx?'
+      - '**.tsx?'
+      - '**.json'
```

Comment:
There are some false positives, like the .json files used by integration tests and spark tests. Can exclude those by adding some negation patterns but not sure if its worth it.